lib/asyncbox.js

```diff
@@ -87,9 +62,7 @@ function nodeifyAll (promiseyMap) {
 }
 
 function asyncify (fn, ...args) {
-  fn.call(null, ...args).then(() => {}, (err) => {
-    console.error(err.stack);
-  });
+  B.resolve(fn.call(null, ...args)).done();
```

Comment:
Just reports the error, I don't think it is possible to throw in the calling context, there are cases where it doesn't exists.